src/keys_handler.rs

```diff
@@ -280,13 +280,22 @@ mod tests {
     };
     use std::env;
     use std::fs;
-    use std::path::Path;
+    use std::path::{Path, PathBuf};
     use std::sync::Arc;
 
     #[cfg(feature = "testing")]
     async fn test_u_or_v_key(key_len: usize, payload: Option<&[u8]>) {
         let test_config = KeylimeConfig::default();
-        let quotedata = web::Data::new(QuoteData::fixture().unwrap()); //#[allow_ci]
+        let mut fixture = QuoteData::fixture().unwrap(); //#[allow_ci]
+
+        // Create temporary working directory and secure mount
+        let temp_workdir = tempfile::tempdir().unwrap(); //#[allow_ci]
+        fixture.secure_mount =
+            PathBuf::from(&temp_workdir.path().join("tmpfs-dev"));
+        fs::create_dir(&fixture.secure_mount).unwrap(); //#[allow_ci]
```

Comment:
I added this because when the secure mount path is passed directly to `run_encrypted_payload()`, the directory is expect to exist (since the secure mount should be ready from the beginning).

In the previous implementation the `secure_mount` would be created by `secure_mount::mount()`. It took me some time to understand what was happening to make the test to fail.